src/substrait/textplan/tests/ParseResultMatchers.h

```diff
@@ -9,20 +9,20 @@
 
 namespace io::substrait::textplan {
 
-[[maybe_unused]] ::testing::Matcher<const ParseResult&> ParsesOk();
```

Comment:
The style here matches that of the rest of the Googletest matchers.  If it's easier to do, we can use our consistent case but I'd prefer to keep these in line if possible.  For instance, mixing whenSerialized with HasSubstr.

whenSerialized(HasSubstr("text"))

Reply:
Not easy to do that, we have to add ` // NOLINT`  for all of them to keep in line with google test matchers.  I saw your other's PR all follow google test style.  if we have to do that, then we  don't need to enforce and change the function case style back to `aNy_Case`, what's your opinion?